tools/flaky-test-retryer/github_commenter.go

```diff
@@ -75,12 +121,19 @@ func (gc *GithubClient) PostComment(jd *JobData, outliers []string) error {
 	if err != nil {
 		return err
 	}
-	oldEntries, err := parseEntries(oldComment)
-	if err != nil {
-		return err
+	oldEntries := make(map[string]*entry)
+	if oldComment != nil {
+		// Only read old entries if it SHA matches with what's currently in this comment
+		if testNameFromComment := regexp.MustCompile(reTestIdentifier).FindStringSubmatch(*oldComment.Body); len(testNameFromComment) < 2 || testNameFromComment[1] == jd.Refs[0].Pulls[0].SHA {
```

Comment:
put on next line :)

Reply:
my golint formatter magically removed the line break, I'll give it another shot

Reply:
it allows me to wrap between conditions